src/renderers/WebGLRenderTarget.js

```diff
@@ -89,6 +89,7 @@ class WebGLRenderTarget extends EventDispatcher {
 		this.viewport.copy( source.viewport );
 
 		this.texture = source.texture.clone();
+		this.texture.image = { ...this.texture.image }; // See #20328.
```

Comment:
Shouldn't it be: 
```js
this.texture.image = { ...source.texture.image }; // See #20328.
```

Reply:
Or better to avoid #22618:
```js
this.texture.image = { 
    width: source.texture.image.width,
    height: source.texture.image.height,
    depth: source.texture.image.depth
};
```

Reply:
👍

Reply:
> ```js
> this.texture.image = { ...source.texture.image }; // See #20328.
> ```

This is the same as `{ ...this.texture.image }` since after `source.texture.clone()` the `source` values are the same object between the two instances.

I don't have an issue with changing it since it's the only location in core that's using spread. Though there are quite a few others it looks like in the examples files.

@Mugen87 suggestion looks good but `Object.assign( {}, this.texture.image )` would work, as well